app/views/proposals/detail/_observer_row.html.haml

```diff
@@ -0,0 +1,5 @@
+%li.observer-list-item
+  = user.full_name
+  = "(#{role_str})" if role_str
```

Comment:
although it creates more lines, I usually prefer to avoid trailing conditionals in ruby bc they are easy to miss!

thoughts on changing this to:

```ruby
- if role_str.present?
  = "(#{role_str})"
```